amazonsqs/publisher.go

```diff
@@ -21,6 +22,10 @@ type PublisherConfig struct {
 }
 
 func NewPublisher(config PublisherConfig, logger watermill.LoggerAdapter) (*Publisher, error) {
+	awsEndpoint :=  os.Getenv("AWS_ENDPOINT")
```

Comment:
It seems there will be some common logic for configuring both publisher and subscriber, so maybe we could move this to a separate function. Or if that's overkill for now, maybe it's enough to introduce a constant with the environment variable name.

On a side note, it looks like the file is not `go fmt`d, can you try running it on the project? :)

Reply:
yup